src/utils/httpRequestParser.ts

```diff
@@ -21,7 +22,8 @@ enum ParseState {
 export class HttpRequestParser implements RequestParser {
     private readonly defaultMethod = 'GET';
     private readonly queryStringLinePrefix = /^\s*[&\?]/;
-    private readonly inputFileSyntax = /^<\s+(.+?)\s*$/;
```

Comment:
The `documentLInkProvider.ts` also uses the same regex to identify the file link, you also need to update the regex there since the Click link feature is broken when appendig `@` after `<`

Reply:
very good point, noticed the underscore stopped working after my change!